src/pages/components/category-card/categorycard.css

```diff
@@ -0,0 +1,41 @@
+.category-card{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+    border-radius: 2rem;
+    border: 1px solid rgba(0, 0, 0,0.5);
+    padding: 2rem;
+    width: 250px;
```

Comment:
Use rem over here

Reply:
i've been told for larger distances use px over rems so using this.